src/messenger-shortcuts.js

```diff
@@ -148,7 +149,8 @@ function toggleInfo() {
 }
 
 function getSearchBar() {
-  return getByAttr('input', 'type', 'text');
+  return document.querySelector('input[type="text"][placeholder="Search Messenger"]');
```

Comment:
Oops this was an old code when I was trying to figure out a solution, I'll take that out.